src/components/TodoItem.js

```diff
@@ -0,0 +1,50 @@
+import React, {useRef} from 'react';
+import {AiFillEdit} from 'react-icons/ai';
+import { IoCheckmarkDoneSharp, IoClose } from 'react-icons/io5';
+import styles from '../styles/main.scss';
+
+const TodoItem = (props) => {
+  const {item, updateTodo, removeTodo, completeTodo} = props;
+
+  const inputRef = useRef(true);
+
+  const changeFocus = () => {
+    inputRef.current.disabled = false;
+    inputRef.current.focus();
+  };
+
+  const update = (id, value, e) => {
+    if (e.which === 13) {
+      //here 13 is key code for enter key
+      updateTodo({id, item: value});
+      inputRef.current.disabled = true;
+    }
+  };
+
+  return (
+    <li key={item.id} className={styles.card}>
+      <textarea
+        ref={inputRef}
+        disabled={inputRef}
+        defaultValue={item.item}
+        onKeyPress={(e) => update(item.id, inputRef.current.value, e)}
+      />
+      <div className={styles.Btns}>
+        <button onClick={() => changeFocus()}>
```

Comment:
changeFocus is without arguments, so we can just write
<button onClick={changeFocus}